react/features/base/connection/actions.web.js

```diff
@@ -8,6 +8,11 @@ import './reducer';
 const JitsiConferenceEvents = JitsiMeetJS.events.conference;
 const logger = require('jitsi-meet-logger').getLogger(__filename);
 
+export {
+    connectionEstablished,
```

Comment:
@lyubomir Do we have some convention about this? I'm OK with @saghul 's proposal but I remembered that generally the other team members were concerned about the number of files that we are creating/using.